compare-bug-version-and-git-branch.sh

```diff
@@ -23,6 +23,9 @@ BUG=$(git show --name-only --format=email | awk '{IGNORECASE=1} /^BUG:/{print $2
 if [ -z "${BUG}" -a "${GERRIT_TOPIC}" = "rfc" ]; then
     echo "No BUG id for rfc needed."
     exit 0
+elif [ -a "${GERRIT_TOPIC#*rfc*}" != "${GERRIT_TOPIC}" -a "${GERRIT_BRANCH}" != "master" ]; then
```

Comment:
Not sure what we're testing here, because this would be caught above.

Reply:
Ack to @nigelbabu's comment.

The intention (as I understand) is to be able to port features to branches without a BUG ID, which is (sort of) disallowed. The exception being between branch creation and first build (typically 2 weeks from branch creation), for noted feature exceptions.

For such noted exceptions, it is better to have a bug created for the backport, as that aids tracking what gets into the release from the point of branching (of course this is when referring to bug tracking alone, and can be achieved when using issue tracking as well, so small proce to pay).

@amarts can you clarify?